app/src/main/java/com/android/shelfLife/ui/newProfile/ProfileScreen.kt

```diff
@@ -42,11 +42,10 @@ import com.example.compose.ThemeMode
 @Composable
 fun ProfileScreen(
     navigationActions: NavigationActions,
-    invitationRepository: InvitationRepository,
-    userRepository: UserRepository,
     context: Context,
+    profileViewModel: ProfileScreenViewModel = hiltViewModel()
 ) {
-  val profileViewModel = viewModel { ProfileScreenViewModel(invitationRepository, userRepository) }
+  Log.d("ProfileScreen", profileViewModel.hashCode().toString())
```

Comment:
Please at least when we submit the final M3 we should really try do delete any log statements we have left in the code.